client/app/views/settings_modal.coffee

```diff
@@ -48,14 +55,15 @@ module.exports = class SettingsModals extends BaseView
             # bound to behaviours managed by Bootsrap.
             @remove()
 
+
     # Close the modal when key `ESCAPE` is pressed.
-    hideOnEscape: (e) ->
-        # escape from outside a datetimepicker
-        @close() if e.which is 27 and not e.isDefaultPrevented()
+    hideOnEscape: (event) =>
+        @close() if 27 in [event.which, event.keyCode]
 
 
     # Close the modal.
-    close: -> @$el.modal 'close'
+    close: ->
+        @$el.modal 'hide'
```

Comment:
I haven't checked the API. Is the change from "close" to "hide" something you wanted or is it a typo? Just to be sure!